lms/static/scripts/frontend_apps/components/test/LMSFilePicker-test.js

```diff
@@ -301,6 +411,36 @@ describe('LMSFilePicker', () => {
     assert.calledWith(onSelectFile, file);
   });
 
+  it('does not invoke `onSelectFile` if chosen file is empty', async () => {
+    const onSelectFile = sinon.stub();
+    const wrapper = renderFilePicker({ onSelectFile });
+    await fakeApiCall;
```

Comment:
I would suggest to wrap up the logic related to waiting for the initial fetch to complete into a helper. In that helper I would recommend watching for the visual output of when loading has complete, rather than waiting exactly one microtask (which is what calling `await` on a non-Promise value does).